tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -377,6 +381,7 @@ public static int setDateOrder(int dateOrder) {
       // ResultSet ---------------------------------------------------------
     $RESULT_SET_FETCH_ROWNUM("tajo.resultset.fetch.rownum", 200),
```

Comment:
Yes, you are right. Too big rows will cause OOM. If we find proper default row number, it will result in better compression ratio.